src/lib.rs

```diff
@@ -193,7 +193,7 @@ pub trait Config: 'static + Send + Sync {
     type State: Clone + Send + Sync;
 
     /// The address type which identifies the remote clients
-    type Address: Clone + PartialEq + Eq + Hash + Send + Sync;
+    type Address: Debug + Clone + PartialEq + Eq + Hash + Send + Sync + Debug;
```

Comment:
While it is debatable if the trait interface should have Debug at all, it definitely does not need it twice :)

Reply:
Haha, I was hoping you would only spot one of them :P